package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "checkif.js",
-    "version": "0.0.1",
+    "version": "0.3.1",
```

Comment:
The `package-lock.json` should not be updated. Please pull the repository before applying your changes